docs/data/date-pickers/date-range-picker/ResponsiveDateRangePicker.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import dayjs from 'dayjs';
-import { DemoContainer, DemoItem } from 'docsx/src/modules/components/DemoContainer';
+import { DemoContainer, DemoItem } from '@mui/x-date-pickers/demo/DemoContainer';
```

Comment:
We probably want  to import from `'@mui/x-date-pickers/demo'`
Imports of depth 3 are not supported by our bundling strategy so it might not work depending on how codesandbox and/or Stackblitz import it (or if users want to import it).

Reply:
CodeSandbox as well as StackBlitz does not seem to have issues with it. 🤔 
And maybe it's even better that users wouldn't be able to easily import it? 😆 🙈

Reply:
If we consider it to be private API, maybe name it `internal-demo` or something like that ? 
To avoid people complaining when we break it.
If it work on both website I guess it's fine to keep the current depth.

If we consider it to be public API, then I'd go for a reduced depth.

Reply:
So, maybe put it in `internals/demo` and re-export at `internals` level? 🤔

Reply:
If you re-export at `internals` level it will be bundled on the pro package.
But put it in `internals/demo` is fine for me.

Reply:
Moved to `internals/demo/` and re-exported to be available at `internals/demo` 😉